ros1/gazebo/README.md

```diff
@@ -0,0 +1,45 @@
+# ROS1 Gazebo simulation demo
+
+Code reference for ROS 1 Gazebo simulation tutorial at https://foxglove.dev/blog/simulating-robotic-scenarios-with-gazebo-and-ros1.
+
+This catkin package contains launch and URDF files to run an example [Gazebo](https://gazebosim.org/) simulation consisting of a robotic arm (UR5e) with a robot mounted depth camera and a separate RGB camera. [Foxglove Studio](https://foxglove.dev/docs/studio) is used to visualize the robot model and other sensor data such as depth & RGB images.
+
+![simulation screenshot](./assets/simulation.png)
+
+## Building this package
+
+To build this package, you will need an existing [ROS installation](http://wiki.ros.org/noetic/Installation).
+
+1. Create a [catkin workspace](https://catkin-tools.readthedocs.io/en/latest/quick_start.html#initializing-a-new-workspace)
+2. Clone this repository in the workspace's `src` folder
+3. Install dependencies with `rosdep install -i --from-path src`
+4. Build the package with `catkin build fg_gazebo_example`
+5. Source the workspace
+
+## Running the simulation
+
+To run the simulation, simply launch the `simulation.launch` launch file:
+```sh
+roslaunch fg_gazebo_example simulation.launch
+```
+
+This will start Gazebo and spawn the robot, the fixed camera and a box-shaped object in the Gazebo world. Additionally, [foxglove_bridge](https://index.ros.org/p/foxglove_bridge/#noetic) is launched which allows ROS topics such as the camera image topics to be visualized with Foxglove Studio.
```

Comment:
Would remove the bulk of these instructions that overlap with the blog post, since we don't have multiple sources of truth.